server/chat-plugins/wifi.tsx

```diff
@@ -767,24 +816,33 @@ export class GTS extends Rooms.RoomGame {
 	stopDeposits() {
 		this.noDeposits = true;
 
-		this.room.send(`|html|<p style="text-align:center;font-size:11pt">More Pokémon have been deposited than there are prizes in this giveaway and new deposits will not be accepted. If you have already deposited a Pokémon, please be patient, and do not withdraw your Pokémon.</p>`);
+		this.room.send(Chat.html`|html|${<p style={{textAlign: 'center', fontSize: '11pt'}}>
+			More Pok&eacute;mon have been deposited than there are prizes in this giveaway and new deposits will not be accepted. {}
+			If you have already deposited a Pok&eacute;mon, please be patient, and do not withdraw your Pok&eacute;mon.
+		</p>}`);
 		this.changeUhtml(this.generateWindow());
 	}
 
 	end(force = false) {
 		if (force) {
 			this.clearTimer();
-			this.changeUhtml('<p style="text-align:center;font-size:13pt;font-weight:bold;">The GTS giveaway was forcibly ended.</p>');
+			this.changeUhtml(
+				<p style={{textAlign: 'center', fontSize: '13pt', fontWeight: 'bold'}}>The GTS giveaway was forcibly ended.</p>
+			);
 			this.room.send("The GTS giveaway was forcibly ended.");
 		} else {
 			this.clearTimer();
-			this.changeUhtml(`<p style="text-align:center;font-size:13pt;font-weight:bold;">The GTS giveaway has finished.</p>`);
+			this.changeUhtml(
+				<p style={{textAlign: 'center', fontSize: '13pt', fontWeight: 'bold'}}>The GTS giveaway has finished.</p>
+			);
 			this.room.modlog({
 				action: 'GTS FINISHED',
 				userid: this.giver.id,
 				note: `their GTS giveaway for "${this.summary}"`,
 			});
-			this.send(`<p style="text-align:center;font-size:11pt">The GTS giveaway for a "<strong>${Utils.escapeHTML(this.lookfor)}</strong>" has finished.</p>`);
+			this.send(Chat.html`${<p style={{textAlign: 'center', fontSize: '11pt'}}>
```

Comment:
`this.send` should probably take a VNode so you don't have to do the weird wrap.

Reply:
Yeah, that was an oversight.